chain/chain/src/chain.rs

```diff
@@ -793,6 +793,7 @@ impl Chain {
     fn get_state_sync_info(
         &self,
         me: &Option<AccountId>,
+        prev_prev_hash: &CryptoHash,
```

Comment:
mini nit: I slightly dislike mixing parameters type (hash & block), would it make any sense to pass, hash, prev_hash and prev_prev_hash as separate arguments?

Reply:
Looking below, the other signature is:
```
header: &BlockHeader
prev_hash: CryptoHash,
prev_prev_hash: CryptoHash,
```

And maybe specify in the comment that header is the epoch first

Reply:
ya agreed, just changed them all to hashes and epoch IDs